index.js

```diff
@@ -572,7 +578,7 @@ class Plugin extends MiniAccountsPlugin {
           })
         })
         .catch((e) => {
-          debug(`failed to pay account.
+          this._log.debug(`failed to pay account.
```

Comment:
If this throws then our claim was probably invalid. It'd be good to see it as an `error`